src/libAtomVM/globalcontext.c

```diff
@@ -212,7 +212,7 @@ int globalcontext_insert_atom_maybe_copy(GlobalContext *glb, AtomString atom_str
 
 bool globalcontext_is_atom_index_equal_to_atom_string(GlobalContext *glb, int atom_index_a, AtomString atom_string_b)
 {
-    AtomString atom_string_a = (AtomString) valueshashtable_get_value(glb->atoms_ids_table, atom_index_a, NULL);
+    AtomString atom_string_a = (AtomString) valueshashtable_get_value(glb->atoms_ids_table, atom_index_a, 0UL);
```

Comment:
I think that types for valueshashtable are just wrong, I think I will change them.